rectools/models/implicit_als.py

```diff
@@ -90,6 +90,26 @@ def _fit(self, dataset: Dataset) -> None:  # type: ignore
                 self.verbose,
             )
 
+    def _fit_partial(self, dataset: Dataset) -> None:
```

Comment:
We've discussed it a lot. Let's not do ALS support for `fit_partial` for now. 
Main reasons are:
1. ALS wrapper has different behaviour then LightFM wrapper in current implementation. ALS cannot do `fit_partial` on model that has not yet been fitted. We can't leave it as it is. 
2. Features support should be here for ALS. So we're gonna have to rewrite some of our code for features support in `fit` to add support for `fit_partial` and also for `epochs` parameter. And it's a long story. It should be in another PR.